i18n/es.jsonp.js

```diff
@@ -178,6 +178,11 @@ document.localeJson = {
       "dialog-readzim-error-message": "Error al leer archivo",
       "dialog-reset-warning-title": "¡AVISO!",
       "dialog-reset-warning-message": "¡Se restablecerá la aplicación al estado original, borrando todas las cachés y configuraciones!",
+      "dialog-metadata-warning": "Atención: ¡los datos anteriores pueden ser falsificados!",
+      "dialog-metadata-name": "Nombre: ",
+      "dialog-metadata-creator": "Creador: ",
+      "dialog-metadata-publisher": "Editor: ",
+      "dialog-metadata-scraper": "Rascador: ",
```

Comment:
The correct word is usually "Raspador" for a Web scraper. "Rascador" means something like "paint scraper".